docs/sources/operators-guide/configuring/reference-configuration-parameters/index.md

```diff
@@ -2717,6 +2717,12 @@ The `limits` block configures default and per-tenant limits imposed by component
 # CLI flag: -ingester.active-series-custom-trackers
 [active_series_custom_trackers_config: <map of tracker name (string) to matcher (string)> | default = ]
 
+# (experimental) Allow samples to be this old for out-of-order. Supported units:
```

Comment:
what does `old` mean in this context? does it mean:

a) older than wall clock time
b) older than the newest ingested sample of this tenant
c) older than the newest ingested sample of this ingester

Reply:
Replied here https://github.com/grafana/mimir/pull/2139#discussion_r903204684